types/openid-client/index.d.ts

```diff
@@ -78,13 +87,17 @@ export class Client {
 
     introspect(token: string, tokenTypeHint?: string, extras?: { readonly introspectBody?: {} }):
         Promise<{ readonly [name: string]: {} | null | undefined }>;
+
+    refresh(refreshToken: string | TokenSet): Promise<TokenSet>;
 }
 
 export class TokenSet {
     readonly access_token?: string;
     readonly token_type?: string;
     readonly id_token?: string;
     readonly refresh_token?: string;
+    readonly expires_at?: number;
+    readonly scope?: string;
```

Comment:
`expires_at`: Okay.

But `scope` ?

See https://github.com/panva/node-openid-client/blob/master/docs/README.md#new-tokensetinput

Reply:
`scope` is a Microsoft specific required parameter and is the rationale for this PR.  Without it I was unable to use the types as is.

https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-protocols-oidc#send-the-sign-in-request

> This request is similar to the first leg of the OAuth 2.0 authorization code flow, with these important distinctions:
> 
> The request must include the openid scope in the scope parameter.
> The response_type parameter must include id_token.
> The request must include the nonce parameter.

Reply:
You linked a `.../oauth2/v2.0/authorize?..scope=..` URL => so the `AuthorizationUrlParameters` input applies (where `scope` already exists). I don't quite get why you are extending the `TokenSet` (which is primarily an output type).

... also please add test cases for your changes.

Thanks!